dagster.yaml

```diff
@@ -2,9 +2,8 @@ scheduler:
   module: dagster.core.scheduler
   class: DagsterDaemonScheduler
 
-run_coordinator:
-  module: dagster.core.run_coordinator
-  class: QueuedRunCoordinator
+run_queue:
+  max_concurrent_runs: 1
```

Comment:
dagster docs are kind of vague on this but i believe that instead of using the `run_coordinator` we can use a queue and then have max current runs. This seems to have the desired output. If I try to launch two jobs only one runs at a time